package.json

```diff
@@ -17,7 +17,7 @@
     "start": "NODE_ENV=production node ./src/server/server.babel.js",
     "dev": "NODE_ENV=development node ./src/server/server.babel.js",
     "build": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\"",
-    "bs": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\" \"npm run start\"",
+    "bs": "npm run build && npm start",
```

Comment:
Channeling your brain – I can see how you were looking at this redundancy and it just bothered the heck out of you didn't it? 👅